FilmFestivalLoader/IFFR/run_iffr_unit_tests.py

```diff
@@ -36,6 +38,10 @@ def __init__(self, title, url, minutes):
         self.description = None
         self.article = None
         self.sorted_title = None
+        self.film_inf0 = None
+
+    def film_info(self, data):
+        return None
```

Comment:
Is this used at all?
At least please do not use a variable name `film_inf0`.

Reply:
Renamed into `film_info`.
And yes, `film_info()` is used, in a method of class `AzPageParser`, which is called in the unit test initialization.